pkg/authorization/authorizer/authorizer.go

```diff
@@ -489,6 +489,7 @@ func GetBootstrapPolicy(masterNamespace string) *authorizationapi.Policy {
 			Name:              authorizationapi.PolicyName,
 			Namespace:         masterNamespace,
 			CreationTimestamp: util.Now(),
+			UID:               util.NewUUID(),
```

Comment:
yes, this is what generates what is stored in etcd.  its buggy because it is not validated yet, and goes direct to registry.